src/lancie-admin-content.html

```diff
@@ -43,11 +44,12 @@
       <lancie-admin-teams data-route="teams" data-path="lancie-admin-endpoint/lancie-admin-pages/lancie-admin-teams.html"></lancie-admin-teams>
       <lancie-admin-types data-route="types" data-path="lancie-admin-endpoint/lancie-admin-pages/lancie-admin-types.html"></lancie-admin-types>
       <lancie-admin-seats data-route="seats" data-path="lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats.html"></lancie-admin-seats>
+      <lancie-admin-hptext data-route="text" data-path="lancie-admin-endpoint/lancie-admin-pages/lancie-admin-hptext.html"></lancie-admin-hptext>
       <lancie-admin-rfid-assign data-route="rfidassign" data-path="lancie-admin-rfid/lancie-admin-rfid-assign.html"></lancie-admin-rfid-assign>
       <lancie-admin-rfid-alcohol data-route="alcoholcheck" data-path="lancie-admin-rfid/lancie-admin-rfid-alcohol.html"></lancie-admin-rfid-alcohol>
       <lancie-admin-rfid-consumption data-route="consumptions" data-path="lancie-admin-rfid/lancie-admin-rfid-consumption.html"></lancie-admin-rfid-consumption>
 
-      <lancie-admin-404 data-route="404" data-path="lancie-admin-404.html"></lancie-404>
+      <lancie-admin-404 data-route="404" data-path="lancie-admin-404.html"></lancie-admin-404>
```

Comment:
🤔

Reply:
Yeah that one has been fixed in 4 branches now 🤔